Updated Sorting and Filtering Feature

mattlee2021 opened the pull request:
Now, users can sort or filter the page by simply clicking on the fields of their choice. The submit buttons have been removed. I simply added a new function called OneClickSubmit() in updateAndSubmit.js to create the new sorting and filtering behavior.

mattlee2021 committed:
Updated Sorting and Filtering Features

Now, users can sort or filter by simply clicking on the fields of their choice. The submit button for sorting and filtering has been removed.

mattlee2021 committed:
Commented out console.log in LaunchesContainer

jsidd reviewed (changes requested):
I am not seeing any major issues.

The only thing I did notice which is also to the issue I created https://github.com/mattlee2021/spaceX_Launches_React/issues/3
is a lack of consistency in the coding style. I think introducing automatic linting will help.

Sometimes your methods start with a capital letter `OneClickSubmit` though most of the time they do not. 
In general Components are capitalized but methods are camelCase.

mattlee2021 committed:
Refactored according to JS Standard

Refactored code to the JS Standard styling guide. To automate styling, I installed the JS Standard extension and run the auto formatting commands.

mattlee2021 committed:
Delete Textbox.js

Deleted unusued file Textbox.js. NotesBox and Textbox are code for the same note taking feature.

jsidd commented:
@mattlee2021 looks much better, thank you.
